### ISSUE TITLE
Review species groups for plants (and generally)

### ISSUE THREAD

**Issue by charvolant** [AtlasOfLivingAustralia/bie-index#276]
The `speciesgroup.json` configuration is old and may no longer reflect the taxonomy in the BIE.

**Comment by vjrj** [comment]
Related: https://atlaslivingaustralia.slack.com/archives/CCTFGEU1G/p1531828275472902

**Comment by ansell** [comment]
When this review occurs it should include the `ozatlas-proxy` hardcoded list:

 https://github.com/AtlasOfLivingAustralia/ozatlas-proxy/blob/master/grails-app/services/mobileauth/GroupService.groovy#L43-L245

**Comment by adam-collins** [comment]
`grailsApplication.config?.species?.groups?.location` should support a URL so that https://biocache.ala.org.au/ws/explore/hierarchy can be used.

Also need to check that it matches in both applications. 
- https://github.com/AtlasOfLivingAustralia/ala-install/blob/master/ansible/roles/biocache-properties/files/subgroups.json
- https://github.com/AtlasOfLivingAustralia/bie-index/blob/master/grails-app/conf/speciesGroups.json

**Comment by adam-collins** [comment]
From #104 
Hi ALA people,
Can you please adjust your species categories to include woodlice Crustacea, Isopoda (I had to submit it in the Crab & Lobster subgroup).
Also there is no category for worms and leeches etc. Annelida, Hirudinea or Oligochaeta etc. I had to a submit a record of a local Striped Forest Leech (Chtonobdella limbata) as a Fish & Carpet Shark! (otherwise the submission was not accepted).
Also, I couldn't find a category for slime moulds etc. (Protista) either.
I intend to edit submissions when you have updated these groupings.
Many thanks
Keep up the great work.
Terry Annable

**Comment by adam-collins** [comment]
Another file for reference, although I do not think this is used https://github.com/AtlasOfLivingAustralia/ala-install/blob/7bcdc90954a1f506483f4514bca046cd49bac1f4/ansible/roles/collectory/files/data/data/taxa/taxa.json

**Comment by adam-collins** [comment]
pipelines uses the namematching service for species group information. This is sourced from these 2 files https://github.com/AtlasOfLivingAustralia/ala-install/tree/master/ansible/roles/namematching-service/files

Everything else should be based on these 2 files.

**Comment by adam-collins** [comment]
Found some related issues in ala-namematching-service
https://github.com/AtlasOfLivingAustralia/ala-namematching-service/issues/67, contemplates additions
https://github.com/AtlasOfLivingAustralia/ala-namematching-service/issues/52, changes the format of `subgroups.json` so it now differs from ala-install and all other applications that use this same file?

**Comment by adam-collins** [comment]
ala-install is now consistent with ala-namematching-service master branch. https://github.com/AtlasOfLivingAustralia/ala-install/pull/759

**Comment by peggynewman** [comment]
We've had a support ticket come in asking about species groups (in this case in the Explore your Area)... "it would also be helpful if the output included such information as the nature of each species (tree, shrub, spider, etc in layman’s language) and whether it is a native or imported species."

**Comment by adam-collins** [comment]
Details of the changes required can be found [here](https://github.com/AtlasOfLivingAustralia/ux-ui/issues/162)
